main.go

```diff
@@ -106,7 +110,7 @@ func main() {
 			defaultMaxConcurrentReconciles = defaultMaxWorkers
```

Comment:
@joelanford, 

A breaking change in the flag --operator-namespace .

Reply:
See my previous comment: https://github.com/joelanford/helm-operator/pull/1#discussion_r443538441

TL;DR: `--operator-namespace` is not a flag of the old helm-operator, nor is it a flag in this refactoring. All we need to make sure to continue supporting is the `WATCH_NAMESPACE` environment variable, which is handled by `manager.ConfigureWatchNamespaces`.